docs/content/customization/generic/global-mirror.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Global Image Registry Mirror"
++++
+
+Add containerd image registry mirror configuration to all Nodes in the cluster.
+
+When the `globalImageRegistryMirror` variable is set, `files` with configurations for [Containerd default mirror](https://github.com/containerd/containerd/blob/main/docs/hosts.md#setup-default-mirror-for-all-registries) will be added.
+
+This customization will be available when the
+[provider-specific cluster configuration patch]({{< ref "..">}}) is included in the `ClusterClass`.
+
+## Example
+
+To provide image registry mirror with CA certificate, specify the following configuration:
+
+If your registry mirror requires a private or self-signed CA certificate,
+create a Kubernetes Secret with the `ca.crt` key populated with the CA certificate in PEM format:
+
+```shell
+kubectl create secret generic my-mirror-ca-cert-secret \
```

Comment:
nit -- we know it's a Secret :-) 
```suggestion
kubectl create secret generic my-mirror-ca-cert \
```

Reply:
https://github.com/d2iq-labs/capi-runtime-extensions/pull/292/commits/6e97e3e261f060c4a57836f5a64cc7af40fe219e